docs/tutorials/ViT-wave-function.ipynb

```diff
@@ -0,0 +1,3423 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        x = x.transpose(0, 1, 3, 2, 4)             # [L_eff, L_eff, b, b]
</pre><p>As you're dragging in einops, you could even use <code>einops.rearrange(x, "b</code>Ly by Lx bx -&gt;b Ly Lx by bx") which is more clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/2004/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>